EXAMPLE/cluster_defs/aws/cluster_vars__cloud.yml

```diff
@@ -24,3 +24,8 @@ cluster_vars:
 #      ports: ["{{ prometheus_node_exporter_port | default(9100) }}"]
 #      group_name: ["{{buildenv}}-private-sg"]
 #      rule_desc: "Prometheus instances attached to {{buildenv}}-private-sg can access the exporter port(s)."
+
+
+# Some of the regions are not available for aws module boto.ec2. If the region definitely exists, you may need to upgrade boto or extend with endpoints_path
+# Adding the below to ec2 invocation fixes the issue
+aws_endpoint_url: https://ec2.{{region}}.amazonaws.com
```

Comment:
Suggestion - add it under the cluster_vars heirarchy?  i.e. `cluster_vars.aws_endpoint_url`

Reply:
Done in https://github.com/sky-uk/clusterverse/pull/122/commits/7e6715301960f5aa89ac059d98ba8b774438d97b, thanks for the suggestion 👍